app/src/main/java/com/poupa/vinylmusicplayer/ui/activities/SettingsActivity.java

```diff
@@ -140,7 +144,13 @@ public void onCreatePreferences(Bundle bundle, String s) {
             addPreferencesFromResource(R.xml.pref_lockscreen);
             addPreferencesFromResource(R.xml.pref_audio);
             addPreferencesFromResource(R.xml.pref_playlists);
-            addPreferencesFromResource(R.xml.pref_blacklist);
+
+            // set summary for whitelist, in order to indicate start directory
+            final String strSummaryWhitelist = getString(R.string.pref_summary_whitelist);
+            final File startDirectory = PreferenceUtil.getInstance().getStartDirectory();
+            final String startPath = FileUtil.safeGetCanonicalPath(startDirectory);
+            findPreference("whitelist_enabled").setSummary(strSummaryWhitelist+startPath);
```

Comment:
You can avoid copying the string literal here - instead use the `PreferenceUtil.WHITELIST_ENABLED` constant

Reply:
Thanks and I forgot this constant completely. Fixed.